src/javascript/static/pages/job_details.js

```diff
@@ -30,7 +31,16 @@ const JobDetails = (() => {
         $sidebar.setVisibility(0);
         $('#title').find('h1').setVisibility(0);
         $('#image').find('img').setVisibility(0);
-        // show section
+
+        window.location.hash = ($sidebar_dept.length > 0 && window.location.hash === '')
+            ? $sidebar_dept.find('a')[0].hash
```

Comment:
I remember we don't want to use something like this, we want to have a list of valid hash, because of the injection security concern raised by Shashank. so we decided to simply leave it blank rather than dynamically show content.